agent/src/issuers/nederlanden.ts

```diff
@@ -0,0 +1,377 @@
+      expiry_date: photoIdPayload.expiry_date.toISOString(),
+      vct: photoIdSdJwt.vct,
+      portrait: `data:image/jpeg;base64,${erikaPortrait.toString('base64')}`,
+    },
+    disclosureFrame: {
+      _sd: [
+        // First payload
+        'family_name_unicode',
+        'family_name_latin1',
+        'given_name_unicode',
+        'given_name_latin1',
+        'birth_date',
+        'portrait',
+        'issue_date',
+        'expiry_date',
+        'issuing_authority_unicode',
+        'issuing_country',
+        'age_over_18',
+        'age_in_years',
+        'age_birth_year',
+        'portrait_capture_date',
+        'birthplace',
+        'name_at_birth',
+        'resident_address_unicode',
+        'resident_city_unicode',
+        'resident_postal_code',
+        'resident_country',
+        'sex',
+        'nationality',
+        'document_number',
+
+        // Second payload
+        'person_id',
+        'birth_country',
+        'birth_state',
+        'birth_city',
+        'administrative_number',
+        'resident_street',
+        'resident_house_number',
+        'travel_document_number',
+
+        // Third payload
+        'dg1',
+        'dg2',
+        'sod',
+      ],
+    },
+  },
+} satisfies StaticSdJwtSignInput
+
+// ======= EU PID 1.5 =======
+
+const eudiPidDisplay = {
+  locale: 'en',
+  name: 'EU PID 1.5',
+  text_color: '#525C75',
+  background_color: '#EBF1F3',
+  background_image: {
+    url: '',
+    uri: '',
+  },
+} satisfies CredentialConfigurationDisplay
+
+const eudiPidPayload = {
+  family_name: 'Mustermann',
+  given_name: 'Erika',
+  birth_date: new DateOnly('1986-03-14'),
+  birth_place: 'The Netherlands, Leiden',
+  nationality: ['NL'],
+  resident_address: 'De Heyderweg 2, Leiden',
+  resident_country: 'NL',
+  resident_state: 'Zuid-Holland',
+  resident_city: 'Leiden',
+  resident_postal_code: '2314 XZ',
+  resident_street: 'De Heyderweg',
+  resident_house_number: '2',
+  personal_administrative_number: '9876543210',
+  portrait: new Uint8Array(erikaPortrait),
+  family_name_birth: 'Mustermann',
+  given_name_birth: 'Erika',
+  sex: 2,
+  email_address: 'erika@mustermann.nl',
+  mobile_phone_number: '+31717993005',
+  expiry_date: new DateOnly('2030-01-28'),
+  issuing_authority: 'Fime',
+  issuing_country: 'NL',
+  document_number: '0123456789',
+  issuance_date: new DateOnly('2025-01-28'),
+  age_over_18: true,
+  age_in_years: 38,
+  age_birth_year: 1986,
+}
+
+export const eudiPidMdoc = {
+  format: OpenId4VciCredentialFormatProfile.MsoMdoc,
+  cryptographic_binding_methods_supported: ['cose_key'],
+  cryptographic_suites_supported: [JwaSignatureAlgorithm.ES256],
+  scope: 'eudi-pid-mdoc',
+  doctype: 'eu.europa.ec.eudi.pid.1',
+  display: [eudiPidDisplay],
+  proof_types_supported: {
+    jwt: {
+      proof_signing_alg_values_supported: [JwaSignatureAlgorithm.ES256],
+    },
+  },
+} as const satisfies MdocConfiguration
+
+export const eudiPidMdocData = {
```

Comment:
For all these credentials, you need to update the presentation during issuance flow:
- For PID credentials add the new credential configuration ids: https://github.com/animo/openid4vc-playground-funke/blob/ed096e6625fae0635f8313905a4e0f30c43b3148/agent/src/issuer.ts#L130-L131
- Then handle all values here: https://github.com/animo/openid4vc-playground-funke/blob/ed096e6625fae0635f8313905a4e0f30c43b3148/agent/src/issuer.ts#L241

Reply:
I think i did this right.